browser/brave_browser_process_impl.cc

```diff
@@ -526,20 +525,13 @@ brave::BraveFarblingService* BraveBrowserProcessImpl::brave_farbling_service() {
   return brave_farbling_service_.get();
 }
 
-misc_metrics::MenuMetrics* BraveBrowserProcessImpl::menu_metrics() {
+misc_metrics::ProcessMiscMetrics*
+BraveBrowserProcessImpl::process_misc_metrics() {
 #if !BUILDFLAG(IS_ANDROID)
-  if (!menu_metrics_) {
-    menu_metrics_ = std::make_unique<misc_metrics::MenuMetrics>(local_state());
+  if (!process_misc_metrics_) {
```

Comment:
It looks like privacy hub metrics which is used on Android has been moved into process misc metrics. But we seem to initialize process misc_metrics_ only when it's not ANDROID. Is it okay?

Reply:
no it's not. good catch, thank you